### PR TITLE
docs: add user survey rfc

### LINKED ISSUE #2685: Add in-UI User Survey
Is your feature request related to a problem? Please describe.
Yes, as of now, it doesn’t seem as if there’s any easy way to get user feedback. Adding a user experience survey to DataHub would enable the collection of feedback from users and thus, give insight into the efficacy of DataHub for our respective organizations.  

Describe the solution you'd like
NYT DGE proposes creating a user experience survey which asks users 
1) How’s your experience so far with DataHub? 
2) Were you able to find what you were looking for?
3) Leave us any feedback
[See Figma design here](https://www.figma.com/file/slcjP197EJUthWM5q7od3n/DataHub-Survey?node-id=0%3A1)
The first iteration of this survey would appear at random times, although in the future we plan to use cookies to more strategically time the appearance of the survey. Cookies could be used to keep track of how many times a user has seen the survey, note whether users don’t want to be asked to answer at all, or to strategize what questions to ask.  

Describe alternatives you've considered
Instead of an in UI pop up survey, there could also be a feedback section of DataHub. We are also considering having a persistent tab available, so users can always have the option to click on the tab to take the survey. 

Additional context
Survey answers are planned to be saved in MySQL. The survey questions will be configurable so that different questions of the same type (ie. yes/no, free form, rate on a scale) can be easily swapped out in the frontend. 

Whatever feedback you have is appreciated!

### PR DESCRIPTION
**Pull request by MelindaCardenas**
closes #2685 
## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [x] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

**Comment by shirshanka** [comment]
Rendered version: https://github.com/MelindaCardenas/datahub/blob/patch-1/docs/rfc/active/2775-user-survey.md

File: docs/rfc/active/2775-user-survey.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+
+**Frontend**
+- For a first iteration, there will only be a single survey with three questions which will appear when a button at the bottom of the viewport is clicked (as shown in Figma). Ideally, this button will appear at the bottom of every page. 
+- A form component will capture survey results and execute a GraphQL mutation that will create a “SurveyResponse”. 
+- Questions will be statically configured in a yaml or json config somewhere in the frontend directory, kind of like this. Expected responses will also be included in the config file so it can be clear what will go in the table. Ex: far right smiley (refer to Figma) would be “really good” because that's the value that would be stipulated in the config. 
+
+**Unresolved Questions**
+I’m curious to know people’s opinions on the ERD above.
+- Do people think having a separate questions table makes sense just for record keeping/organization? On one hand, having a questions table allows for more complexity later on if people want to build on top. On the other hand, it might be redundant/unnecessary to have both a questions table and questions config file.
```
Comment (jjoyce0510) [inline docs/rfc/active/2775-user-survey.md:41]: Let's probably start with questions config file only :)

Comment (MelindaCardenas) [reply docs/rfc/active/2775-user-survey.md:41]: sounds good john :)

**Comment by shirshanka** [comment]
Closing as this RFC is technically outdated. Feel free to re-open and petition for inclusion if you believe that the RFC is in good shape to add into the repo. 
/cc @jjoyce0510

**Pull request closed without merging**